easybuild/easyconfigs/p/ParallelIO/ParallelIO-2.2.2a-intel-2017a.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'ParallelIO'
+version = '2.2.2a'
+
+homepage = 'https://github.com/NCAR/ParallelIO'
+description = """A high-level Parallel I/O Library for structured grid applications"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+## underscore = "${version//./_}"           something like this  ??
+## sources = ['pio%(underscore)s.tar.gz']   bash sub in easyconf ??
+sources = ['pio2_2_2a.tar.gz']
+source_urls = ['https://github.com/NCAR/ParallelIO/archive/']
```

Comment:
We've recently added a requirement to include checksums for all sources.

Please add an `SHA256` checksum here.